plugins/woocommerce/client/legacy/css/woocommerce.scss

```diff
@@ -2382,3 +2382,15 @@ body:not(.search-results) .twentysixteen .entry-summary {
 	background: inherit;
 	color: inherit;
 }
+
+/**
+ * Twenty Twenty Five specific styles
+ */
+body.theme-twentytwentyfive {
```

Comment:
I don't think we should add the `twentytwentyfive` specific CSS to the general `woocommerce.scss` file.

It looks like that we usually create a new `scss` file for the theme specifically and only enqueue that style when that theme is active.
See the example of twenty twenty three:
css file: https://github.com/woocommerce/woocommerce/blob/c5faf64bfab9a87e24b10ba805a1090741a40537/plugins/woocommerce/client/legacy/css/twenty-twenty-three.scss#L1
PHP file to enqueue the style: https://github.com/woocommerce/woocommerce/blob/0f204dbb575fed81d9a0ed8befc0e83a5b992cfa/plugins/woocommerce/includes/theme-support/class-wc-twenty-twenty-three.php#L27
Case statement for only including the above PHP file when the specific theme is active: https://github.com/woocommerce/woocommerce/blob/f107670192f02eb6d4a36181f067f413ec1a8ea1/plugins/woocommerce/includes/class-woocommerce.php#L796